### PR TITLE
Explorer and RPC provider updates for Filecoin

### PR DESCRIPTION
**Pull request by eshon**

### CODE REVIEW

**Comment by Oluwarichee** [comment]
Let set it up please 💡

File: _data/chains/eip155-314.json [hunk 1]
```diff
@@ -31,8 +35,8 @@
       "standard": "EIP3091"
     },
     {
-      "name": "Filmine",
-      "url": "https://explorer.filmine.io",
```
Comment (ligi) [inline _data/chains/eip155-314.json:35]: why do we remove this one and not just add the other one?

Comment (eshon) [reply _data/chains/eip155-314.json:35]: They are rebranding

**Review by raulk — APPROVED** [review]
LGTM.

**Pull request closed without merging**